pkg/services/pluginsettings/service.go

```diff
@@ -21,6 +21,12 @@ type Service struct {
 	pluginSettingDecryptionCache secureJSONDecryptionCache
 }
 
+type Store interface {
```

Comment:
I got confused about naming. If this is the interface for `*Service` I think this interface should be named `Service` and register wire to bind service implementation struct  to `Service` interface.  Doing this it would probably make sense to create a new service package which holds the implementation as can be seen in other places, e.g. https://github.com/grafana/grafana/tree/main/pkg/services/dashboardimport/service.

Reply:
I can call the struct a ServiceImpl and add it to wire